proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -886,6 +886,7 @@ message GenerateLastHistoryReplicationTasksRequest {
 
     string namespace_id = 1;
     temporal.api.common.v1.WorkflowExecution execution = 2;
+    string target_cluster_id = 3;
```

Comment:
Could we have a list of target clusters?

Reply:
the cluster id means the initial failover version. either use it with the IFV or rename to cluster name